src/main/services/api.ts

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+
+export default class Api {
+  private baseUrl = process.env.API_URL;
+  private apiKey = process.env.API_KEY;
+  private headers;
+
+  constructor(baseUrl?: string, apiKey?: string) {
+    if (baseUrl) {
+      this.baseUrl = baseUrl;
+    }
+    else if (!this.baseUrl) {
+      throw new Error('No base url supplied and no default set in environment variables');
+    }
+
+    if (apiKey) {
+      this.apiKey = apiKey;
+    }
+    else if (!this.apiKey) {
+      throw new Error('No api key supplied and no default set in environment variables');
+    }
+
+    this.headers = {
+      'Content-Type': 'application/json',
+      'Authorization': `Bearer ${this.apiKey}`,
+      'Accept': 'application/json',
+      'UserId': ''
+    };
+  }
+
+  private async request(type: string, endpoint: string, userId: string, data?: any) {
+    let {headers} = this;
+    headers.UserId = userId;
+    return axios({
```

Comment:
can leave this for the time being to get us going but before we go live lets refactor this so we can pass in an `AxiosInstance` - it'll make testing much easier